backend/.env.example

```
@@ -1,187 +1,76 @@
 # =============================================================================
 
 # -----------------------------------------------------------------------------
 # OPENAI CONFIGURATION (Primary LLM Provider)
 # -----------------------------------------------------------------------------
-# Get your API key from: https://platform.openai.com/api-keys
-# Required for story generation using GPT models
-OPENAI_API_KEY=your_openai_api_key_here
-OPENAI_MODEL=gpt-4o-mini
+OPENAI_API_KEY=
+OPENAI_MODEL=gpt-5-mini
 OPENAI_ENDPOINT=https://api.openai.com/v1/chat/completions
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The OPENAI_ENDPOINT key should go before the OPENAI_MODEL key

(UnorderedKey)